src/PockyBot.NET/Services/Triggers/Peg.cs

```diff
@@ -77,6 +85,9 @@ public async Task<Message> Respond(Message message)
             var receiver = await _chatHelper.People.GetPersonAsync(receiverId).ConfigureAwait(false);
             var dbReceiver = _pockyUserRepository.AddOrUpdateUser(receiver.UserId, receiver.Username);
 
+            _logger.LogInformation(
+                "Giving peg with comment {comment}, sender {senderId}, receiver {receiverId}, validity {isPegValid}",
```

Comment:
better IMO to put the longest part (the comment) *last*. Makes it easier to ignore in favour of the probably more relevant information.

Reply:
Nice catch. updated